### PR TITLE
rename mod process to io and clippy refactorings

### PR DESCRIPTION
**Pull request by philipcraig**
* rename mod `process` to `io`
* rename `files` to `process_files`
* move the `use` to top of module, which is where it begins its scope
* remove raw non-ascii literals in code
* iterate more iodiomatically

### CODE REVIEW

File: src/io.rs [hunk 1]
```diff
@@ -86,12 +86,12 @@ mod tests {
     #[test]
     #[should_panic(expected = "Failed to open file at /made/up/file/location")]
     fn missing_file_causes_a_panic() {
-        files(vec!["/made/up/file/location".to_string()]).unwrap();
+        process_files(vec!["/made/up/file/location".to_string()]).unwrap();
```
Comment (jbcoe) [inline src/io.rs:89]: super-nit but this _could_ pass. Can we make it truly unpassable?

**Review by jbcoe — DISMISSED** [review]

File: src/io.rs (hunk 1 above)
Comment (philipcraig) [reply src/io.rs:89]: > super-nit but this _could_ pass. Can we make it truly unpassable?

but how? an illegal path?

Comment (jbcoe) [reply src/io.rs:89]: Pretty sure it will never come up.

'/dev/null/no/such/path'

Could work?

**Review by jbcoe — APPROVED** [review]

File: src/io.rs (hunk 1 above)
Comment (philipcraig) [reply src/io.rs:89]: ok I will merge this PR and fix in a future merge

**Pull request merged**